# Message describing single TopoNavNode

int64 node_id
time last_updated
time last_pose_updated
int64 area_id
geometry_msgs/Pose pose
bool is_door
